deploy/eks/cloudbeat-ds.yml

```diff
@@ -330,3 +336,12 @@ roleRef:
   name: cloudbeat-kubeadm-config
   apiGroup: rbac.authorization.k8s.io
 ---
+# TODO: Support local env clusters with multiple cloudbeats
```

Comment:
this todo needs to be more specific (preferably, open a task for it). This is also relevant for all other ymls todos on this PR

Reply:
I wrote the comment in order to emphasize the gap for local clusters.
Do we want to cover the local cluster leader election for standalone cloudbeat pods?
If so, then I will open a gap.

Reply:
Opened https://github.com/elastic/cloudbeat/issues/162 and edited the code to refer to that issue.